### PR TITLE
feat: Add Analytics for attach images - MEED-2435 - Meeds-io/MIPs#53

### PR DESCRIPTION
**Pull request by SaraBoutej**
This PR allows to add information about the module and the sub module for attach image analytics

### CODE REVIEW

File: portlets/src/main/resources/locale/portlet/Analytics_en.properties [hunk 1]
```diff
@@ -17,6 +17,7 @@ analytics.field.label.announcementActivityId=Announcement Activity Id
 analytics.field.label.announcementAssignee=Announcement assignee
 analytics.field.label.announcementComment=Announcement comment
 analytics.field.label.announcementComment=Announcement comment
+analytics.field.label.objectType= Gamification: Object type
```
Comment (boubaker) [inline portlets/src/main/resources/locale/portlet/Analytics_en.properties]: collision with the label defined in Tasks, the same key with two different labels.

File: services/src/main/java/io/meeds/gamification/analytics/RuleAttachmentAnalyticsListener.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  @Override
+  protected void extendStatisticData(StatisticData statisticData, ObjectAttachmentId objectAttachment) {
+    RuleDTO rule = ruleService.findRuleById(Long.parseLong(objectAttachment.getObjectId()));
+    if (rule != null) {
+      ProgramDTO program = rule.getProgram();
+      if (program != null) {
+        statisticData.addParameter("programTitle", rule.getProgram().getTitle());
+      }
+      statisticData.addParameter("ruleTitle", rule.getTitle());
+      statisticData.addParameter("objectType", "rule");
```
Comment (boubaker) [inline services/src/main/java/io/meeds/gamification/analytics/RuleAttachmentAnalyticsListener.java]: `rule` should be used in submodule only

```suggestion
```

File: portlets/src/main/resources/locale/portlet/Analytics_en.properties (hunk 1 above)
Comment (SaraBoutej) [reply portlets/src/main/resources/locale/portlet/Analytics_en.properties]: done!

File: services/src/main/java/io/meeds/gamification/analytics/RuleAttachmentAnalyticsListener.java (hunk 2 above)
Comment (SaraBoutej) [reply services/src/main/java/io/meeds/gamification/analytics/RuleAttachmentAnalyticsListener.java]: done!

**Review by boubaker — APPROVED** [review]

**Pull request merged**